### PR TITLE
More bits and pieces

### PR DESCRIPTION
**Pull request by andrewhayward**
Mostly to fix stuff for #11.

### CODE REVIEW

File: gameon/base/static/base/css/master.css [hunk 1]
```diff
@@ -82,28 +82,27 @@ div[role=main] .busta {
...
+    background: #EBEBEB 50% 50% no-repeat;
     border-radius: 4px;
     -moz-box-sizing: border-box;
     -webkit-box-sizing: border-box;
     box-sizing: border-box;
-    border: 1px solid #666;
-    display: block;
+    border: 1px inset rgba(255,255,255,0.5);
+}
+.mini-col.header-block.judges::before {
```
Comment (rossbruniges) [inline gameon/base/static/base/css/master.css]: You could use the existing page IDs for this

File: gameon/base/static/base/css/master.css [hunk 2]
```diff
@@ -82,28 +82,27 @@ div[role=main] .busta {
...
+    background-image: url(/static/base/img/css/judges.png);
+}
+.mini-col.header-block.rules::before,
+.mini-col.header-block.resources::before {
```
Comment (rossbruniges) [inline gameon/base/static/base/css/master.css]: Likewise - this feels like a case for the page IDs

File: gameon/base/static/base/css/master.css (hunk 1 above)
Comment (rossbruniges) [inline gameon/base/static/base/css/master.css]: Looking at things a bit more I also don't think we need the 'header-block' class on these. There is only one mini-col on the pages that we're using it so can just use that.

File: gameon/base/static/base/css/master.css (hunk 2 above)
Comment (rossbruniges) [inline gameon/base/static/base/css/master.css]: Looking at things a bit more I also don't think we need the 'header-block' class on these. There is only one mini-col on the pages that we're using it so can just use that.

**Comment by rossbruniges** [comment]
You might want to rebase in from master here - that might still try and bring the commit history over but it shouldn't re-apply the files again. Failing that grab the file changes, revert master to before and start this stuff over again....

**Comment by andrewhayward** [comment]
Closing this mess - starting again.

**Pull request closed without merging**